src/com/engeto/CountryStats.java

```diff
@@ -0,0 +1,54 @@
+package com.engeto;
+
+import java.util.ArrayList;
+
+public class CountryStats {
+    private ArrayList<Country> highestVAT;
+    private ArrayList<Country> lowestVAT;
+
+    public void highestVAT(ArrayList<Country> countries) {
```

Comment:
Je vhodnejsie pouzit pre parametre interface nez konkretnu triedu (tj List namiesto ArrayList) - v podstate ti nezalezi na konkretnej implementacie Listu, ale na tom, aby to splnal kontrat polozene v interface List (tj objekt sa chova ako List).